libc/benchmarks/gpu/CMakeLists.txt

```diff
@@ -15,13 +15,30 @@ function(add_benchmark benchmark_name)
   endif()
   add_libc_hermetic(
     ${benchmark_name}
-    IS_BENCHMARK
+    IS_GPU_BENCHMARK
     LINK_LIBRARIES
       LibcGpuBenchmark.hermetic
       ${BENCHMARK_LINK_LIBRARIES}
     ${BENCHMARK_UNPARSED_ARGUMENTS}
   )
   get_fq_target_name(${benchmark_name} fq_target_name)
+  set(fq_build_target_name ${fq_target_name}.__build__)
+
+  # We want to dump kernel resource usage for GPU benchmarks
+  get_target_property(gpu_loader_exe libc.utils.gpu.loader "EXECUTABLE")
+  set(res_usage_cmd $<$<BOOL:${LIBC_TARGET_OS_IS_GPU}>:${gpu_loader_exe}>
+    ${CMAKE_CROSSCOMPILING_EMULATOR}
```

Comment:
What's this do here? I was actually looking into working with this at some point, since it will allow me to test the other targets in the future.

Reply:
I think I misinterpreted what we talked about before and made `nvptx-loader --print-resource-usage` only print and disable running the kernel - This was like the 2nd cmd that you run for `nvptx-loader` after launching the kernel just to see the register usage